calico_node/startup/startup.go

```diff
@@ -115,10 +87,16 @@ func main() {
 		terminate()
 	}
 
-	// Set other Felix config that is not yet in the node resource.
-	if err := ensureDefaultConfig(client, node); err != nil {
-		fatal("Unable to set global default configuration: %s", err)
-		terminate()
+	// Configure IP Pool configuration.
```

Comment:
I know you didn't write it.. but this comment could use some improving.

It's about as useful as this right now:

```
// configureIPPools
configureIPPools()
```